projects/pgai/tests/vectorizer/cli/test_vectorizer_core.py

```diff
@@ -487,3 +492,59 @@ def test_regression_vectorizer_uuid_pk_with_error(
         cur.execute("SELECT count(*) FROM ai.vectorizer_errors")
         results = cur.fetchall()
         assert results[0]["count"] == 1
+
+
+def test_vectorizer_without_retries_works_as_expected(
+    cli_db: tuple[PostgresContainer, Connection],
+    cli_db_url: str,
+    vcr_: Any,
+):
+    """Test that vectorizers work as intended without
+    the loading_retries feature enabled"""
+    _, connection = cli_db
+    table_name = setup_source_table(connection, 1)
+
+    # Given a vectorizer and the feature `loading_retries` disabled.
+    vectorizer_id = configure_vectorizer(
+        table_name,
+        cli_db[1],
+    )
+    features = Features("0.9.0")
```

Comment:
note: this will have to change with the next rebase

Reply:
fixed, thanks